driver-mapping/src/main/java/com/datastax/driver/mapping/ReflectionUtils.java

```diff
@@ -25,12 +26,19 @@
 import java.lang.reflect.*;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * Utility methods related to reflection.
  */
 class ReflectionUtils {
 
+    private static final Set<String> EXCLUDED_PROPERTIES = ImmutableSet.of(
```

Comment:
One limitation with a system property is that you only control the list globally. Someone might need a per-mapper configuration.
Yet another way would be a property on the `Table` and `UDT` annotations.